libraries/AP_NavEKF3/AP_NavEKF3_OptFlowFusion.cpp

```diff
@@ -167,90 +163,111 @@ void NavEKF3_core::EstimateTerrainOffset()
             }
         }
 
-        if (fuseOptFlowData && !cantFuseFlowData) {
+        if (!cantFuseFlowData) {
```

Comment:
Should we put an else if instead here? I'm asking because previously this would only fuse optical flow data if rangefinder data wasn't available, but now it will fuse both (provided the parameter is set to use it for height).

Reply:
I would prefer to be able to use flow for height, even if range finder is available given how intermittent range finder data can be. I am planning to get test data for that combination in the next two weeks.

Reply:
See 
https://github.com/ArduPilot/ardupilot/pull/10539/commits/1eae4592aad00955e8ee5ab0a95b85a03e342b38 and 
https://github.com/ArduPilot/ardupilot/pull/10539/commits/447a0f257764e4a2420f1f96a2c33ec45b5a5dc2